### ISSUE TITLE
Some additional app features

### ISSUE THREAD

**Issue by jprybylski** [jprybylski/deckifyr#23]
Seems fairly low-hanging fruit for the app to:

- [ ] have add/remove slide functionality, where remove comes with a clear warning about the permanence. Add should have a quick layout (or empty except for furniture) selector that comes up on add.
- [ ] have a table in the editor to edit layouts, for sections to be dragged around (this would ideally update slides using those layouts, if possible)
- [ ] A little box showing the color in a color input. A color picker might be nice if easy, but this would be a safe compromise.

After these features are implemented, it would be reasonable to update the vignette to ensure its persistent accuracy
